### PR TITLE
Table performance paging

### PR DESCRIPTION
**Pull request by jvigliotta**
partial-close: #7268 (items 1)

### Describe your changes:

- added the ability to configure tables to be in "performance" mode (limited number of rows shown, defaults to 50, but configurable)
- configurable persistability
- check for exporting to CSV (must be in "unlimited" mode)
- table row collections handle a majority of the work by limiting the rows to the configured or default amount and holding onto an additional amount (same as configured/default) to be used if an item is removed to fill in any missing rows
- realtime and fixed time are supported, in realtime it's a rolling window of the specified row limit
- at most will be 2x the limit of rows per endpoint

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Is this a breaking change to be called out in the release notes?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?

### CODE REVIEW

**Review by charlesh88 — APPROVED** [review]
Added a bit of bondo and sanded (fixed alignment and button classing, changed button labeling slightly and added a title). LGTM!

**Review by akhenry — APPROVED** [review]

File: src/plugins/telemetryTable/TelemetryTable.js [hunk 1]
```diff
@@ -101,18 +102,40 @@ export default class TelemetryTable extends EventEmitter {
     }
   }
 
+  updateTelemetryMode(mode) {
+    if (this.telemetryMode === mode) {
+      return;
+    }
+
+    this.telemetryMode = mode;
```
[the remark this answers is not in the document]
Comment (unlikelyzero) [reply src/plugins/telemetryTable/TelemetryTable.js:110]: yea on autosave

**Review by unlikelyzero — APPROVED** [review]

**Comment by akhenry** [comment]
Created https://github.com/nasa/openmct/issues/7419 for followup.

File: src/plugins/telemetryTable/TelemetryTable.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (davetsay) [reply src/plugins/telemetryTable/TelemetryTable.js:110]: thats how i would have wrote it... hehe

**Pull request merged**